.gitignore.orig

```diff
@@ -0,0 +1,48 @@
+*.py[cod]
```

Comment:
ah, we should actually remove this whole file --> it is a merge conflict record (.orig)